src/Microsoft.Data.SqlClient/ref/System.Data.Common.Batch.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Data.Common
```

Comment:
Again, since we are only implementing the new batch APIs in netcore, do we still need this file?

Reply:
removed.